src/main/kotlin/com/openlattice/data/storage/PostgresEntityDataQueryService.kt

```diff
@@ -240,17 +241,14 @@ class PostgresEntityDataQueryService(
             authorizedPropertyTypes: Map<UUID, PropertyType>,
             awsPassthrough: Boolean = false
     ): WriteEvent {
-        return hds.connection.use { connection ->
-            val writeEvent = upsertEntities(connection, entitySetId, entities, authorizedPropertyTypes, awsPassthrough)
-//            connection.commit()
-//            connection.autoCommit = true
-            return@use writeEvent
-        }
+
+        return upsertEntities(entitySetId, null, entities, authorizedPropertyTypes, awsPassthrough)
```

Comment:
Don't use null use a no-op function, which will be elided by the compiler anyway and avoid the null check.